src/main/scala/com/markatta/timeforscala/factories.scala

```diff
@@ -461,3 +461,21 @@ object ZoneOffset {
   def apply(accessor: TemporalAccessor): ZoneOffset =
     java.time.ZoneOffset.from(accessor)
 }
+
+object TimeUnit {
+
+  final val NANOSECONDS: TimeUnit = java.util.concurrent.TimeUnit.NANOSECONDS
```

Comment:
Constants in Scala are usually (and elsewhere in this library) camelcase with a starting uppercase, so this would be `NanoSeconds`.